app/controllers/spaces_controller.rb

```diff
@@ -3,6 +3,7 @@ def index
   end
 
   def show
+    @space = Space.find(space_params[:id])
```

Comment:
As cleared up earlier, this should be `params[:id]` instead, given it does not involve any POSTed params that we need to sanitize before committing them to the DB.